### ISSUE TITLE
[Fabric] App throws unhandled exception when Assistive Technology is enabled during app launch.

### ISSUE THREAD

**Issue by chiaramooney** [microsoft/react-native-windows#11929]
### Problem Description

On Fabric, when an app is launched while Assistive Technology is enabled, the app hits an unhandled exception causing the app to become unresponsive. This behavior should not occur. App users should be able to launch RNW apps with AT enabled before app launch. 

Note this error occurs in cases when screen readers or Accessibility Insights are launched and when trying to launch a fabric app in automated testing (which makes use of AT to launch the app). 

Offending line: https://github.com/microsoft/react-native-windows/blob/07df14f4f5da2088a2a596c189cd4d3726831af6/packages/playground/windows/playground-composition/Playground-Composition.cpp#L494

Culprit: The call to UiaProvider() ultimately tries to get data from the FabricUIManager which is not initialized at the time this code is called. 

Code added: https://github.com/microsoft/react-native-windows/pull/11129/files#diff-c8de480fc16d4e24df921c34b116cbd653f776278c95b3512d8c5fe1a02d909c

### Steps To Reproduce

1. Enable Narrator.
2. Launch playground-composition app. 

### Expected Results

App should launch with no errors. 

### CLI version

npx react-native -v

### Environment

```markdown
npx react-native info
```

### Target Platform Version

None

### Visual Studio Version

None

### Build Configuration

None

**Comment by FalseLobster** [comment]
I thought I addressed this issue already, but it should be an easy fix
